### PR TITLE
Merge release/6.0-maui branch back into release/6.0

### PR DESCRIPTION
**Pull request by akoeplinger**

### CODE REVIEW

**Comment by akoeplinger** [comment]
This was already approved as https://github.com/dotnet/runtime/pull/64352.

**Comment by steveisok** [comment]
This change now has a blocking dependency on https://github.com/dotnet/runtime/pull/64749.  In addition, we'll need to publish 100 and 200 workload manifests - that will come in a PR shortly.

**Comment by steveisok** [comment]
PR has been updated to produce a 100 and 200 mono toolchain manifest workloads.  Once https://github.com/dotnet/runtime/pull/64749 has been merged, this PR can go through.

File: eng/Versions.props [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <MajorVersion>6</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>3</PatchVersion>
-    <SdkBandVersion>6.0.100</SdkBandVersion>
+    <SdkBandVersion>6.0.200</SdkBandVersion>
```
Comment (lewing) [inline eng/Versions.props:9]: should this be removed now?

**Comment by steveisok** [comment]
Merged in latest arcade bump. After CI runs, this is good to go.

File: eng/Versions.props (hunk 1 above)
Comment (steveisok) [reply eng/Versions.props:9]: No, it's still needed by the wasm tests.  We can fix that up in another PR.

**Comment by steveisok** [comment]
Official build looks good... It includes the 100 and 200 manifest msi's.

https://dev.azure.com/dnceng/internal/_build/results?buildId=1599661&view=results

**Review by steveisok — APPROVED** [review]

**Comment by safern** [comment]
Failures are known. I need to put up a PR to fix them.

**Pull request merged**